torch/csrc/jit/runtime/register_distributed_ops.cpp

```diff
@@ -149,12 +166,27 @@ RegisterOperators reg_rpc_ops({
                  const string& keyStr = keyIValue.toStringRef();
                  names.emplace_back(keyStr);
                }
-               throw std::runtime_error(functionSchema.findErrorInKwargs(names));
+               throw std::runtime_error(
+                   functionSchema.findErrorInKwargs(names));
+             }
+
+             // Get destination WorkerName.
+             std::string dstWorkerNameStr;
+             if (dstWorkerNameIValue.isString()) {
+               // ivalue::ConstantString::str_ is a const member, which can't be
+               // moved, copy it here.
+               dstWorkerNameStr = dstWorkerNameIValue.toStringRef();
+             } else {
+               TORCH_INTERNAL_ASSERT(dstWorkerNameIValue.isInt());
```

Comment:
this has been checked in the beginning of this function?

Reply:
nice suggestion!